src/components/LibraryListForm/index.tsx

```diff
@@ -110,7 +110,7 @@ const LibraryListForm = ({ libraryList = [] }: LibraryListFormProps) => {
         everything you need from a branch that&apos;s most convenient for you.
         You can change your home library at any point through your account.
       </p>
-      <p>Skipping this step will automatically a home library.</p>
+      <p>Skipping this step will automatically default a home library.</p>
```

Comment:
Still missing a word (sorry!); "default TO", I assume.

Reply:
I'm 99% sure that's ungrammatical (I don't think I've ever heard "default" used transitively), which seems...less-than-ideal...for a library; I know it's not coming from you, but is there someone we can check it with?  (I assume someone else just accidentally left the preposition out...).  @killerpuppytails?

Reply:
The copy has not be finalized and some of this is using quick generalization. Please ignore for now.